src/ds/ds6/ds6-device.cpp

```diff
@@ -448,23 +452,24 @@ namespace librealsense
 
         _color_calib_table_raw = [this]()
         {
-            return get_raw_calibration_table(rgb_calibration_id);
+            return get_ds6_raw_calibration_table(ds6_calibration_table_id::rgb_calibration_id);
         };
 
         if (((hw_mon_over_xu) && (RS400_IMU_PID != _pid)) || (!group.usb_devices.size()))
```

Comment:
`RS400_IMU_PID` not needed.
this is `ds6` file

Reply:
If ok, this will be removed (if needed, I have to check) within the refactoring continuing

Reply:
Sure